src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = [...table.querySelectorAll('tr')];
+
+for (const row of rows) {
+  const newFootTh = document.createElement('th');
+  const tfoot = document.querySelector('tfoot');
+  const newTh = document.createElement('th');
+  const thead = document.querySelector('thead');
```

Comment:
Similar to the `tfoot` query, the `thead` element is being queried inside the loop for each row. Consider moving this query outside the loop for better performance.